<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>779</width>
    <height>793</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GuiTest</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="rightMargin">
         <number>0</number>
        </property>
        <item>
         <widget class="QComboBox" name="cb_context"/>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPlainTextEdit" name="ctx_description"/>
      </item>
     </layout>
    </item>
    <item row="1" column="1">
     <widget class="Line" name="line_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>Editor</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPlainTextEdit" name="editor"/>
    </item>
    <item row="4" column="1">
     <widget class="Line" name="line_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QPushButton" name="btn_execute">
      <property name="text">
       <string>execute</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="topMargin">
       <number>0</number>
      </property>
      <item>
       <widget class="QPushButton" name="btn_blue">
        <property name="text">
         <string>blue</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_red">
        <property name="text">
         <string>red</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_green">
        <property name="text">
         <string>green</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="11" column="1">
     <widget class="Line" name="line">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="12" column="1">
     <widget class="QLabel" name="label">
      <property name="text">
       <string>Output</string>
      </property>
     </widget>
    </item>
    <item row="13" column="1">
     <widget class="QPlainTextEdit" name="output">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="readOnly">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>779</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLoad_file">
    <property name="title">
     <string>Load </string>
    </property>
    <addaction name="actionLoad_file"/>
   </widget>
   <addaction name="menuLoad_file"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionLoad_file">
   <property name="text">
    <string>Load file ...</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
